### PR TITLE
Repo improvements

### PR DESCRIPTION
**Pull request by joselrio**
This PR is for adding several repository improvements such as:
- New workflows;
- New npm script;
- Fix issue due to case sensitive folders names;
- Create LICENSE file;
- Removed docs folder;

### CODE REVIEW

File: LICENSE.txt [hunk 1]
```diff
@@ -0,0 +1,28 @@
+BSD 3-Clause License
+
+Copyright (c) 2023, José Rio
```
Comment (BenOsodrac) [inline LICENSE.txt:3]: Shouldn't here be OutSystems, instead of José Rio?

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
 	"name": "outsystems-ui",
-	"version": "2.15.0",
+	"version": "2.14.0",
```
Comment (BenOsodrac) [inline package.json:3]: version should be 2.15.0

**Review by BenOsodrac — CHANGES_REQUESTED** [review]

File: LICENSE.txt (hunk 1 above)
Comment (joselrio) [reply LICENSE.txt]: Done! Tks Bernardo!

File: package.json (hunk 2 above)
Comment (joselrio) [reply package.json]: Done! Tks Bernardo!

**Review by BenOsodrac — APPROVED** [review]

**Review by gnbm — APPROVED** [review]
LGTM 😎

**Pull request merged**